contrib/root/usr/share/scripts/patroni/post_init.sh

```diff
@@ -2,8 +2,14 @@
 set -Eeu
 
 if [[ (! -z "$APP_USER") &&  (! -z "$APP_PASSWORD") && (! -z "$APP_DATABASE")]]; then
-  echo "Creating user ${APP_USER}"
-  psql "$1" -w -c "create user \"${APP_USER}\" WITH LOGIN ENCRYPTED PASSWORD '${APP_PASSWORD}'"
+
+  if [ "$APP_USER" = "postgres" ]; then
+    echo "Updating postgres user with new password $APP_PASSWORD"
```

Comment:
Should we ever be printing out the incoming password into the logs? It's good for debugging I suppose, but this could be an unnecessary secret leak that may get caught and saved by log aggregators downstream.

Reply:
i totally agree. I felt weird but i noticed patroni had always done that with `psql "$1" -w -c "create user \"${APP_USER}\" WITH LOGIN ENCRYPTED PASSWORD '${APP_PASSWORD}'` 

I should have questioned the statusquo instead of following along

Reply:
Also @jujaga I was thinking the database creation should be in its own control flow statement. Just incase a user was using the default user and password for postgres? You could still create a database?